frontend.js

```diff
@@ -32,12 +32,22 @@ function cleanLineEndings(str) {
 
 function formatDate(d) {
   if (d) {
+    // 14 Jun 2018
     return moment(d).format("DD MMM YYYY");
   } else {
     return "";
   }
 }
 
+function formatDateTime(d) {
+  if (d) {
+    // Sunday, February 14th 2010, 3:25:50 pm
+    return moment(d).format("dddd, MMMM Do YYYY, h:mm:ss a");
```

Comment:
Consider using 24hr clock and removing the am/pm indication, to reduce some confusion. Additionally, consider adding a time zone indication.